src/frontend/pages/Stream/Stream.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import SideBar from "../../components/SideBar/SideBar";
+import ReactPlayer from "react-player";
+import "./Stream.css";
+import { useParams } from "react-router-dom";
+import { useData } from "../../context/data-context";
+
+function Stream() {
+  const { videoID } = useParams();
+  const {state} = useData();
+  const {videos} = state;
+
+  function findVideo(id){
+      const currentVideo = videos.find(video => video._id === id)
+      return currentVideo;
+  }
+
```

Comment:
This can be converted to an arrow function

Reply:
well I prefer this way